src/NowLine/NowLine.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { Text, View, Animated } from 'react-native';
+import PropTypes from 'prop-types';
+
+import {
+  minutesToYDimension,
+  CONTENT_OFFSET,
+} from '../utils';
+import styles from './NowLine.styles';
+
+const UPDATE_EVERY_MILLISECONDS = 60 * 1000; // 1 minute
+
+const getCurrentTop = (hoursInDisplay) => {
+  const now = new Date();
+  const minutes = now.getHours() * 60 + now.getMinutes();
+  return minutesToYDimension(hoursInDisplay, minutes) + CONTENT_OFFSET;
+};
+
+class NowLine extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      currentTop: new Animated.Value(getCurrentTop(this.props.hoursInDisplay)),
+    }
+
+    this.intervalCallbackId = null;
+  }
+
+  componentDidMount() {
+    this.intervalCallbackId = setInterval(() => {
```

Comment:
Hi @pdpino , I just found out the API that may help to loop animation, https://reactnative.dev/docs/animated#loop. And can we set `useNativeDriver` to `true`? Just the suggestions, I'm not sure if it works as expected. In case you don't think it's worth, please skip my comment and we can look back later.

Reply:
That's a good idea! To be honest, `setInterval` feels kind of hacky to me, using `loop` could be a better idea.

I will check it out and let you know

Reply:
I tried out both things:

* I changed to `useNativeDriver: true`: it cannot be used if the animated value points to layout styles, such as `top` (see [animated caveats described](https://reactnative.dev/docs/animations#caveats)). However, it can be used with `transform`, so I changed the logic to setting a fixed `top` value, and animating the `translateY` style.
  **Notice I tested this only on android, not iOS**
* `Animated.loop` does not allow a function call on each iteration, and we need to call a function to update the date (i.e. call `new Date()` to calculate the new position with the current time). I found some ugly workarounds, but I preferred the `setInterval` solution. (See for example [this blog](https://medium.com/@benjamintodts/react-natives-animated-loop-invoking-a-callback-whenever-an-iteration-finishes-1c3581d38d54), which uses a recursive call -> I think it could lead to out-of-stack errors).

I also set `isInteraction: false`, which should give the animation less priority.

Reply:
Thanks @pdpino , I believe this is good enough!